tests/smoke/events.js

```diff
@@ -76,7 +115,25 @@
 				'Events and expected length should be the same. Actual events:\n' + observer.events);
 
 			for (var i = 0; i < events.length; i++) {
-				assert.areSame(expected[i], events[i]);
+				assert.areSame(expected[i], events[i].name);
+			}
+		};
+
+		observer.assertRootIs = function (expectedRoot) {
+			var events = observer.events,
+				event,
+				i,
+				root;
+
+			for (i=0, event=events[i]; i<events.length; i++) {
+				if (event.name === 'spellCheckComplete') {
+					continue;
```

Comment:
there's maybe a longer term change where we actually care about which element we finished checking on.  but that is not for now, where this event doesn't actually pass a rootElement along